src/prisma/schema.prisma

```diff
@@ -38,3 +40,16 @@ model Profile {
   website                  Json?
   tdaGradYearBannerVisible Json
 }
+
+model Post {
+  id                 Int   @id @default(autoincrement())
+  userId             Int?   @unique
+  user               User?  @relation("UserToPost",fields: [userId], references: [id])
+  content            String
+  image              String?
+  likes              Int?      @default(0)
+  dislikes           Int?      @default(0)
```

Comment:
I would make these both (likes and dislikes) required since they are required in the entity, and they will default to 0 anyways, so they will always be there.